<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewsWidget</class>
 <widget class="QWidget" name="ViewsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewsWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QComboBox" name="comboBoxAnnotation">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>--</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelSample">
     <property name="text">
      <string>Sample:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelPath">
     <property name="toolTip">
      <string>Relative path from dataset root directory</string>
     </property>
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEditThumbnail">
     <property name="toolTip">
      <string>Relative path from dataset root directory</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelAnnotation">
     <property name="text">
      <string>Annotation:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButtonThumbnailFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEditPath">
     <property name="toolTip">
      <string>Relative path from dataset root directory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="comboBoxSample">
     <item>
      <property name="text">
       <string>--</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelThumbnail">
     <property name="toolTip">
      <string>Relative path from dataset root directory</string>
     </property>
     <property name="text">
      <string>Thumbnail:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonViewFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPlainTextEdit" name="plainTextEditDescription"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
